src/scripts/main.js

```diff
@@ -1,5 +1,39 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
-});
+
+let wall = document.querySelector('.wall')
+
+let spider = document.querySelector('.spider')
+
+let spiderStyle = window.getComputedStyle(spider)
+
+document.addEventListener('click', (event) => {
```

Comment:
```suggestion
document.addEventListener('click', (e) => {
```
rename event to e to fix error